pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/EACInformation/EACInformation.tsx

```diff
@@ -20,9 +26,35 @@ const EACInformation = ({
         (Application dédiée à la généralisation de l'éducation artistique et
         culturelle).
       </p>
+
+      {isCreatingVenue && !canOffererCreateCollectiveOffer && (
+        <Banner
+          type="attention"
+          href="https://passculture.typeform.com/to/VtKospEg"
+          linkTitle="Faire une demande de référencement"
+        >
+          Pour proposer des informations à destination d’un groupe scolaire,
+          vous devez être référencé auprès du ministère de l’Éducation Nationale
+          et du ministère de la Culture.
+        </Banner>
+      )}
+
+      {isCreatingVenue && canOffererCreateCollectiveOffer && (
+        <Banner
+          type="notification-info"
+          href="https://passculture.typeform.com/to/VtKospEg"
+          linkTitle="Faire une demande de référencement"
+        >
+          Une fois votre lieu créé, vous pourrez renseigner des informations
+          pour les enseignants en revenant sur cette page.
+        </Banner>
+      )}
+
       <ButtonLink
-        to={`/structures/${offererId}/lieux/${venueId}/eac`}
+        to={`/structures/${offererId}/lieux/${venue?.id}/eac`}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Qu'est ce qui se passe si on a pas d'id :raised_eyebrow: 
L'url est KO ? On a une redirection ? ‏

Reply:
L'url est KO mais le seul cas où c'est censé être possible c'est si on est en train de créer un lieu (car on passe un lieu null) mais pas embêtant car le lien est disabled :)